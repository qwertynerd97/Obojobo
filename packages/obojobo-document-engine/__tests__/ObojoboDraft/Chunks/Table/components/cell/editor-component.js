import React from 'react'
import renderer from 'react-test-renderer'

import Cell from 'ObojoboDraft/Chunks/Table/components/cell/editor-component'

describe('Cell Editor Node', () => {
	test('Cell component', () => {
		const component = renderer.create(
			<Cell
				node={{
					data: {
						get: () => {
							return { header: false }
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})

	test('Cell component as header', () => {
		const component = renderer.create(
			<Cell
				node={{
					data: {
						get: () => {
							return { header: true }
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
