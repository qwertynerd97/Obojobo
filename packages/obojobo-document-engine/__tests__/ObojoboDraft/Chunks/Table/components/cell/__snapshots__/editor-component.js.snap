// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell Editor Node Cell component 1`] = `<td />`;

exports[`Cell Editor Node Cell component as header 1`] = `<th />`;
